xtask/src/set_scarb_version.rs

```diff
@@ -0,0 +1,31 @@
+use anyhow::Result;
+use clap::Parser;
+use semver::Version;
+use toml_edit::{value, Document};
+use xshell::{cmd, Shell};
+
+#[derive(Parser)]
+pub struct Args {
+    version: Version,
+    #[arg(long, default_value_t = false)]
+    dry_run: bool,
+}
+
+pub fn main(args: Args) -> Result<()> {
+    let sh = Shell::new()?;
+
+    let mut cargo_toml = sh.read_file("Cargo.toml")?.parse::<Document>()?;
+    let package = cargo_toml["workspace"]["package"].as_table_mut().unwrap();
+
+    package["version"] = value(args.version.to_string());
+
+    eprintln!("[workspace.package]\n{package}");
```

Comment:
Is this intentional?

Reply:
yes, I want to see the changes no matter if it's dry/non-dry mode and I model the output to look almost like what'd be written to a file